packages/appstore/src/components/containers/trading-app-card-actions.tsx

```diff
@@ -28,10 +29,26 @@ const TradingAppCardActions = ({
     is_buttons_disabled,
     is_real,
 }: Actions) => {
+    const {
```

Comment:
@sergei-deriv better to keep this component dummy without any connection to stores, and just add a disabled props to it which gets true/false from the place the component is using in

Reply:
@mahdiyeh-deriv Ok, I'll change it